lib/FieldType/Mapper/AbstractMapper.php

```diff
@@ -0,0 +1,95 @@
+<?php
+/**
+ * This file is part of the eZ RepositoryForms package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ * @version //autogentag//
+ */
+namespace EzSystems\RepositoryForms\FieldType\Mapper;
+
+use eZ\Publish\API\Repository\FieldTypeService;
+use eZ\Publish\API\Repository\Values\ContentType\FieldDefinition;
+use EzSystems\RepositoryForms\Data\Content\FieldData;
+use EzSystems\RepositoryForms\FieldType\DataTransformer\FieldValueTransformer;
+use EzSystems\RepositoryForms\FieldType\FieldTypeFormMapperInterface;
+use Symfony\Component\Form\FormInterface;
+
+abstract class AbstractMapper implements FieldTypeFormMapperInterface
```

Comment:
Hmm, while reading the mapper, I'm thinking that we might wanna reverse the logic here

Instead of having every Mapper extend this one, why don't we make the common behaviour a service that each mapper uses ? It avoids the tight coupling we get with protected methods.